mocca-jaxrs2/build.gradle

```diff
@@ -1,5 +1,11 @@
 dependencies {
     implementation  project(':mocca-client'),
                     lib.feign_jaxrs2
-    api             lib.jaxrs2
+    api             lib.jaxrs2,
+                    lib.slf4j_api
+
+    testImplementation project(':mocca-http-client-tests'),
```

Comment:
These dependencies are not being used. But I suppose that is because this a draft PR, and you plan to add tests to this module. Is that right? Just double checking.